.github/workflows/nightly_test.yml

```diff
@@ -1,7 +1,11 @@
 name: Nightly Test
 
 on:
-  #pull_request:
+  pull_request:
+    branches: [ "main" ]
+    paths:
+      - 'python/orca/**'
+      - '.github/workflows/nightly_test.yml'
```

Comment:
please remove this pull_request action, if you want to trigger this test for PR validation, please update PR_Validation.yml to add Orca-Ray-Ctx-Example.